### PR TITLE
Build grpc ingestor image

### PR DESCRIPTION
**Pull request by jt-dd**
Adding the support of building the ingestor image locally `kubehound dev --ingestor`.

Other fix:
* Fixing the grpc-ingestor image.
* Removing goreleaser (using buildx now for the binary)

### CODE REVIEW

File: deployments/kubehound/docker-compose.ingestor.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+version: "3.8"
```
Comment (edznux-dd) [inline deployments/kubehound/docker-compose.ingestor.yaml:1]: version or not version, this is the question? (you removed it from the other ones, soo....)

**Review by edznux-dd — DISMISSED** [review]

File: deployments/kubehound/docker-compose.ingestor.yaml (hunk 1 above)
Comment (jt-dd) [reply deployments/kubehound/docker-compose.ingestor.yaml]: seems deprecated:
* The top-level version property is defined by the Compose Specification for backward compatibility. It is only informative you'll receive a warning message that it is obsolete if used.

ref: https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-top-level-element-obsolete

removing it

File: deployments/kubehound/docker-compose.datadog.yaml [hunk 2]
```diff
@@ -2,11 +2,9 @@ services:
   datadog:
     image: gcr.io/datadoghq/agent:7-jmx
     restart: unless-stopped
-    profiles: ["infra"]
-    container_name: ${COMPOSE_PROJECT_NAME}-datadog-agent
     ports:
       - "127.0.0.1:8225:8125/UDP"
-      - "127.0.0.1:8226:8126"
+      - "127.0.0.1:8126:8126"
```
Comment (edznux-dd) [inline deployments/kubehound/docker-compose.datadog.yaml]: I think that will need to change so we don't have conflict with the local agent (if any)

File: pkg/backend/project.go [hunk 3]
```diff
@@ -123,10 +127,33 @@ func loadEmbeddedConfig(ctx context.Context) (*types.Project, error) {
...
+	// For local version (when the version is "dirty", using latest to have a working binary)
+	version := map[string]string{"VersionTag": "latest"}
+	if !strings.HasSuffix(config.BuildVersion, "dirty") {
+		version["VersionTag"] = config.BuildVersion
+	}
+
+	if strings.HasSuffix(filepath, ".tpl") {
+		tmpl, err := template.New(filepath).ParseFS(embedconfigdocker.F, filepath)
+		if err != nil {
+			return nil, fmt.Errorf("new template :%w", err)
```
Comment (edznux-dd) [inline pkg/backend/project.go:143]:
```suggestion
			return nil, fmt.Errorf("new template: %w", err)
```

**Review by edznux-dd — DISMISSED** [review]

File: deployments/kubehound/docker-compose.datadog.yaml (hunk 2 above)
Comment (jt-dd) [reply deployments/kubehound/docker-compose.datadog.yaml]: done

**Review by edznux-dd — APPROVED** [review]

**Pull request merged**